Assets/Scripts/Controllers/GameController.cs

```diff
@@ -91,11 +129,34 @@ IEnumerator StartRound()
         _uiController.HideNewRoundDisplay();
         yield return new WaitForSeconds(.3f);
 
-        // Check if we passed 5 and if so reshuffel decks
-        if (_roundNumber >= 5)
+        //Reset both players' hand infos to base stats
+        _player1.ResetCurrentPepemonStats();
+        _player2.ResetCurrentPepemonStats();
+
+        //Refresh cards every 5 turns
+        if (_roundNumber % 5 == 0)
+        {
+            //Need to refresh decks
+
+            // Shuffle player1 support cards
+            _player1.GetAndShuffelDeck(69, _roundNumber, battleSeed);
+
+            //Reset played card count
+            _player1.PlayedCardCount = 0;
+
+            // Shuffle player2 support cards
+            _player2.GetAndShuffelDeck(420, _roundNumber, battleSeed);
```

Comment:
Would be good to see constants for these magic numbers (69, 420, 69420)

Reply:
Nice. Okay

Reply:
Updated in [this](https://github.com/pepemon-dao/pepemon-battle-degen/pull/48/commits/60b768f7be47f29f9a22f5590f431c19447d0f3a) commit